Gruntfile.js

```diff
@@ -21,10 +21,44 @@ module.exports = function( grunt ) {
 				}
 			}
 		},
+
+		checktextdomain: {
+			options:{
+				text_domain: 'wc-admin',
+				keywords: [
+					'__:1,2d',
+					'_e:1,2d',
+					'_x:1,2c,3d',
+					'esc_html__:1,2d',
+					'esc_html_e:1,2d',
+					'esc_html_x:1,2c,3d',
+					'esc_attr__:1,2d',
+					'esc_attr_e:1,2d',
+					'esc_attr_x:1,2c,3d',
+					'_ex:1,2c,3d',
+					'_n:1,2,4d',
+					'_nx:1,2,4c,5d',
+					'_n_noop:1,2,3d',
+					'_nx_noop:1,2,3c,4d'
+				]
+			},
+			files: {
+				src:  [
+					'**/*.php',               // Include all files/
+					'!node_modules/**',       // Exclude node_modules/
+					'!tests/**',              // Exclude tests/
+					'!vendor/**',             // Exclude vendor/
+					'!tmp/**'                 // Exclude tmp/
+				],
+				expand: true
+			}
+		}
 	} );
 
 	// Load NPM tasks to be used here.
 	grunt.loadNpmTasks( 'grunt-wp-i18n' );
+	grunt.loadNpmTasks( 'grunt-checktextdomain' );
+	grunt.registerTask( 'i18n', ['addtextdomain', 'makepot'] );
```

Comment:
`addtextdomain` was removed in #481 since it didn't seem useful for the project, but we can bring it back since we'll have more PHP files now.

The `checktextdomain` task isn't added/called anywhere– is that just for dev workflow, or should that be called in the precommit hook?

Reply:
> addtextdomain was removed in #481 since it didn't seem useful for the project, but we can bring it back since we'll have more PHP files now.

Sorry, I included back by mistake after fixing conflicts.

> The checktextdomain task isn't added/called anywhere– is that just for dev workflow, or should that be called in the precommit hook?

On WooCommerce core we run this task just before release something, so I don't think we need to check it for every commit, maybe we can include as `pre-push`.